.travis.yml

```diff
@@ -1,7 +1,9 @@
 language: node_js
 node_js:
-  - "node"
-  - "4" # Fedora
+  - "0.10" # EL6, Trusty, Jessie
```

Comment:
@gailsteiger see https://travis-ci.org/theforeman/foreman/builds/174192667 (also linked bellow) but it does open the question if its the tests who fail because of old node versions or the actual code (e.g. if webpack:compile passes).

further, I don't know if its an issue to upgrade node/npm just when building the packages for other oss (e.g. non rhel7), /cc @mmoll